modules/monitors/variables.tf

```diff
@@ -1,6 +1,7 @@
 # https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/monitor
 variable "datadog_monitors" {
   type = map(object({
+    enabled                = bool
```

Comment:
If we're going to add a new input, can we set this type to `any` and comment out the keys so we can optionally add enabled or not ? That way we won't be forced to add it for each monitor and the `main.tf` can default to `enabled=true` if it's not provided.

cc: @aknysh what do you think ?

Reply:
yes, `any` is a good idea